src/world.cpp

```diff
@@ -90,7 +90,7 @@ World::World(const char * topo_map, const char * pol_map, const char * div_map)
 
 		// Associate tiles with provinces
 		this->tiles[i].province_id = (size_t)-1;
-		uint n_provinces = provinces.size();
+		unsigned int n_provinces = provinces.size();
```

Comment:
Why not size_type (which is what the size() method returns)?
```suggestion
		size_type n_provinces = provinces.size();
```
In the loop below size_t is used.